application/classes/Ushahidi/Repository/Notification.php

```diff
@@ -39,12 +39,11 @@ protected function getTable()
 	public function setSearchConditions(SearchData $search)
 	{
 		$query = $this->search_query;
-				
-		$user = $this->getUser();
 
-		// Limit search to user's records unless they are admin
-		if (! $this->isUserAdmin($user)) {
```

Comment:
I don't think we should be removing this. Though maybe we should update it to check a permission as well as isUserAdmin()